src/libraries/System.Text.RegularExpressions/gen/UpgradeToRegexGeneratorCodeFixer.cs

```diff
@@ -177,12 +177,12 @@ private static async Task<Document> ConvertToSourceGenerator(Document document,
                 SyntaxNode createRegexMethod = generator.InvocationExpression(generator.IdentifierName(methodName));
                 SyntaxNode method = generator.InvocationExpression(generator.MemberAccessExpression(createRegexMethod, invocationOperation.TargetMethod.Name), arguments.Select(arg => arg.Syntax).ToArray());
 
-                newTypeDeclaration = newTypeDeclaration.ReplaceNode(nodeToFix, method);
+                newTypeDeclaration = newTypeDeclaration.ReplaceNode(nodeToFix, WithTrivia(method, nodeToFix));
             }
             else // When using a Regex constructor
             {
                 SyntaxNode invokeMethod = generator.InvocationExpression(generator.IdentifierName(methodName));
-                newTypeDeclaration = newTypeDeclaration.ReplaceNode(nodeToFix, invokeMethod);
+                newTypeDeclaration = newTypeDeclaration.ReplaceNode(nodeToFix, WithTrivia(invokeMethod, nodeToFix));
```

Comment:
Can we add tests to validate trivia in the fixer?

Reply:
sure, I'll add these in a bit.